docs/modules/ROOT/partials/recovery/recover-etcd.adoc

```diff
@@ -0,0 +1,147 @@
+export TENANT_ID=$(curl -sH "Authorization: Bearer ${COMMODORE_API_TOKEN}" ${COMMODORE_API_URL}/clusters/${CLUSTER_ID} | jq -r .tenant)
+----
+
+.Fetch backup url from cluster repo
+
+[source,bash]
+----
+GIT_REPO=$(curl -sH "Authorization: Bearer ${COMMODORE_API_TOKEN}" ${COMMODORE_API_URL}/clusters/${CLUSTER_ID} | jq -r .gitRepo.url)
+git clone --depth 1 $GIT_REPO cluster-repo
+RESTIC_ENDPOINT=$(find cluster-repo/manifests/cluster-backup -name '*.yaml' -exec yq eval-all 'select(.kind == "Schedule" and .metadata.name == "etcd" ) | .spec.backend.s3.endpoint' {} \;)
+RESTIC_BUCKET=$(find cluster-repo/manifests/cluster-backup -name '*.yaml' -exec yq eval-all 'select(.kind == "Schedule" and .metadata.name == "etcd" ) | .spec.backend.s3.bucket' {} \;)
+export RESTIC_REPOSITORY="s3:${RESTIC_ENDPOINT}/${RESTIC_BUCKET}"
+echo $RESTIC_REPOSITORY
+rm -rf cluster-repo
+----
+
+include::partial$connect-to-vault.adoc[]
+
+.Fetch backup secrets from vault
+
+[source,bash]
+----
+export RESTIC_PASSWORD=$(vault kv get \
+  -format=json "clusters/kv/${TENANT_ID}/${CLUSTER_ID}/cluster-backup" | jq -r '.data.data.password')
+ifeval::["{provider}" == "cloudscale"]
+export AWS_ACCESS_KEY_ID=$(vault kv get \
+  -format=json "clusters/kv/${TENANT_ID}/${CLUSTER_ID}/cloudscale" | jq -r '.data.data.s3_access_key')
+export AWS_SECRET_ACCESS_KEY=$(vault kv get \
+  -format=json "clusters/kv/${TENANT_ID}/${CLUSTER_ID}/cloudscale" | jq -r '.data.data.s3_secret_key')
+endif::[]
+ifeval::["{provider}" == "exoscale"]
+export AWS_ACCESS_KEY_ID=$(vault kv get \
+  -format=json "clusters/kv/${TENANT_ID}/${CLUSTER_ID}/exoscale/storage_iam" | jq -r '.data.data.s3_access_key')
+export AWS_SECRET_ACCESS_KEY=$(vault kv get \
+  -format=json "clusters/kv/${TENANT_ID}/${CLUSTER_ID}/exoscale/storage_iam" | jq -r '.data.data.s3_secret_key')
+endif::[]
+----
+
+.Download files from latest etcd snapshot
+
+[source,bash]
+----
+SNAPSHOT_ID=$(restic snapshots --json --latest=1 --path /syn-cluster-backup-etcd-etcd-backup.tar.gz | jq -r '.[0].id')
+restic dump "${SNAPSHOT_ID}" /syn-cluster-backup-etcd-etcd-backup.tar.gz | tar xzv
+----
+
+== Connect to Master Node by SSH
+
+.Fetch the ssh key
+
+[source,bash,subs="attributes+"]
+----
+vault kv get -format=json clusters/kv/${TENANT_ID}/${CLUSTER_ID}/{provider}/ssh \
+  | jq -r '.data.data.private_key' | base64 --decode > ssh_key
+chmod 400 ssh_key
+----
+
+NOTE: The following steps are VSHN specific
+
+.Find load balancer host
+
+[source,bash]
+----
+LB_HOST=$(ssh management1.corp.vshn.net grep -E "^Host.*${CLUSTER_ID}" '~/.ssh/config' | head -1 | awk '{print $2}')
+echo $LB_HOST
+----
+
+NOTE: Ensure your ssh config is up-to-date: `ssh management1.corp.vshn.net sshop generate -li`.
+
+.Upload recovery files to master node
+
+[source,bash]
+----
+MASTER_NODE=etcd-0
+scp -J management1.corp.vshn.net,"${LB_HOST}" -i ssh_key static_kuberesources_*.tar.gz snapshot_*.db "core@${MASTER_NODE}:"
```

Comment:
Same here, usually it's not necessary to use `management1.corp.vshn.net` as jumphost.